Source/JavaScriptCore/runtime/OptionsList.h

```diff
@@ -563,6 +563,7 @@ bool hasCapacityToUseLargeGigacage();
     v(Bool, exposeCustomSettersOnGlobalObjectForTesting, false, Normal, nullptr) \
     v(Bool, useJITCage, canUseJITCage(), Normal, nullptr) \
     v(Bool, useAllocationProfiling, false, Normal, "Allows toggling of bmalloc/libPAS allocation profiling features at JSC launch."_s) \
+    v(Unsigned, allocationProfilingMode, false, Normal, "Allows custom arguments to be passed to bmalloc/libPAS allocation profiling features at JSC launch."_s) \
```

Comment:
Why is this` false`? shouldn't it be `0`? Also, why do we need this in OptionsList.h at all? Couldn't this and `useAllocationProfiling` just be comments here?

Reply:
Will fix the false/0 bit.
Re.: why it's in OptionsList.h, otherwise `jsc` complains about `ERROR: invalid option: JSC_useAllocationPorifling=1`
This seems like a cleaner solution than hacking the check to overlook these.

Reply:
> Re.: why it's in OptionsList.h, otherwise jsc complains about ERROR: invalid option: JSC_useAllocationProfiling=1
This seems like a cleaner solution than hacking the check to overlook these.

Ah, gotcha, makes sense.